Source/Core/BoxGeometry.js

```diff
@@ -132,7 +179,7 @@ define([
      * @param {Number[]} array The array to pack into.
      * @param {Number} [startingIndex=0] The index into the array at which to start packing the elements.
      */
-    BoxGeometry.pack = function(value, array, startingIndex) {
+    BoxGeometry.pack = function (value, array, startingIndex) {
```

Comment:
There are a lot of new spacing changes in this file and we don't want.

Did WebStorm autoformat these?